From f5e4dbd209b22624de1fffadf4fa613b3b6c1c76 Mon Sep 17 00:00:00 2001
From: Nguyen Anh Minh <minh.nguyen-anh@banvien.com.vn>
Date: Thu, 2 Nov 2023 17:08:10 +0700
Subject: [PATCH] Fix syntas error

---
 drivers/leds/hello.c | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/drivers/leds/hello.c b/drivers/leds/hello.c
index b8c47c8fc166..384c19e96ef4 100644
--- a/drivers/leds/hello.c
+++ b/drivers/leds/hello.c
@@ -21,15 +21,13 @@
 
 int init_module(void)
 {
-	printk("Hello World! I am Minh :333\n
-    This is a module builtin\n");
+	printk("Hello World! I am Minh :333!!! This is a module builtin\n");
 	return 0;
 }
 
 void cleanup_module(void)
 {
-	printk("Goodbye Cruel World! See you again, Minh :333\n
-    The module builtin cleanup\n");
+	printk("Goodbye Cruel World! See you again, Minh :333!!! The module builtin cleanup\n");
 }
 
 module_init(init_module);
-- 
2.25.1

